src/index.ts

```diff
@@ -55,11 +56,17 @@ async function main() {
         branch,
       });
       console.log(`Found ${data.total_count} runs total.`);
+      console.log(data.workflow_runs.map(run => `- ${run.html_url}`).join("\n"));
+      
       const runningWorkflows = data.workflow_runs.filter(
-        run => run.head_branch === branch && run.head_sha !== headSha && run.status !== 'completed' &&
-            new Date(run.created_at) < new Date(current_run.created_at)
+        run => run.head_branch === branch && 
+               (allow_matching_sha || run.head_sha !== headSha) &&
+               run.status !== 'completed' &&
+               new Date(run.created_at) < new Date(current_run.created_at)
       );
-      console.log(`Found ${runningWorkflows.length} runs in progress.`);
+      console.log(`Found ${runningWorkflows.length} runs in to cancel.`);
```

Comment:
```suggestion
      console.log(`Found ${runningWorkflows.length} runs to cancel:`);
```

Reply:
```suggestion
      console.log(`Found ${runningWorkflows.length} runs to cancel.`);
```

Reply:
@styfle I think that we should keep it a dot `.`; as if theirs is 0 runs found, that wouldn't make much sense:
```
Found 2 runs to cancel:
- http...
- http...
Other logs
```
vs 

```
Found 0 runs to cancel:
Other logs
```

Reply:
Let's remove the additional `console.log()`, its not needed for this PR